src/components/ViewAdmin.vue

```diff
@@ -258,10 +258,10 @@
 				</NcCheckboxRadioSwitch>
 			</p>
 			<p>
-				{{ t('recognize', 'Like most machine learning models, recognize will run even faster when using a GPU. Setting this up is non-trivial but works well when everything is setup correctly.') }}
+				{{ t('recognize', 'Like most machine learning models, Recognize will run even faster when using a GPU. Setting this up is non-trivial but works well when everything is setup correctly.') }}
```

Comment:
I think the lowercase was on purpose originally, but I realize it's confusing. :+1: